md/rest-api-authorization.md

```diff
@@ -34,45 +34,63 @@ You need to [add authorization to your custom pages](#migrate) before you activa
+If there are conflicts between permissions, the more restrictive definition overrides the more general.
+You can see this in the example above, where the specific permission for user 3 overrides the general permission for seeing user information.
+
+::: warning
+Do **not** modify file `resources-permissions-mapping.properties` directly, as it is reserved for default values.
+Custom values should be added manually in file `resources-permissions-mapping-`**`-custom`**`.properties`
+:::
 
 #### Compound permissions mapping
 
-The `compound-permissions-mapping.properties` file defines sets of simple permissions that are grouped together into a compound permission.
+The `compound-permissions-mapping-*.properties` files define sets of simple permissions that are grouped together into a compound permission.
 You can use a compound permission as "shorthand" for a list of simple permissions.
-By default, the file contains a compound permission that corresponds to each page of the Bonita BPM Portal, including [custom pages](#custom_pages).
+By default, the file `resources-permissions-mapping.properties` contains a compound permission that corresponds to each page of the Bonita BPM Portal,
+including [custom pages](#custom_pages).
 
 For example: `userlistingadmin=[profile_visualization, process_comment, organization_visualization, tenant_platform_visualization, organization_management]`
 
 This specifies the REST API permissions that are granted with the Bonita BPM Portal Administrator page that lists all the users in the tenant.
 
 By default, there is a compound permission defined for each page in the standard Bonita BPM Portal and there is also one for each provided custom page.
 
-When you install a custom page in the portal, if the page declares its resources properly, then it will add a line in this file. Then all the users being able to access this page (because it is part of a custom profile or Living Application they have access to) will also be automatically granted the necessary permissions to call the required REST API resources.
+When you install a custom page in the portal, if the page declares its resources properly, then a new compound permission will be added in an internal version
+of this file (`compound-permissions-mapping-internal.properties`). Then all the users being able to access this page (because it is part of a custom profile or
+Living Application they have access to) will also be automatically granted the necessary permissions to call the required REST API resources.
+
+::: warning
+Do **not** modify file `compound-permissions-mapping.properties` directly, as it is reserved for default values.
+Custom values should be added manually in file `compound-permissions-mapping`**`-custom`**`.properties`
+:::
+
 
```

Comment:
I know this is not related to what you did, but maybe we could add another warning:
"If the page declares resources provided by a REST API extension (link to the REST API extention page), then the REST API extension must be deployed before the page, otherwise the compound permissions won't be automatically created when deploying the page."